app/common/services/translation.service.js

```diff
@@ -42,20 +42,31 @@ function (
         });
     };
 
-    var getLanguage = function () {
+    var getLanguage = function (config) {
```

Comment:
All works fine but I not sure I understand this config. I cannot find anywhere getLanguage is called with a config?